spec/pom.xml

```diff
@@ -21,14 +21,10 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>a2a-java-sdk-common</artifactId>
         </dependency>
-
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
-        </dependency>
         <dependency>
-            <groupId>com.fasterxml.jackson.datatype</groupId>
-            <artifactId>jackson-datatype-jsr310</artifactId>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
the Jackson dependencies should also be removed from the parent pom's `<dependencyManagement>`